<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Find people</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>1</number>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
       <item row="0" column="0">
        <widget class="PeopleWidget" name="peopleWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Find activities</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>1</number>
       </property>
       <property name="spacing">
        <number>1</number>
       </property>
       <item row="0" column="0">
        <widget class="ActivitiesWidget" name="activitiesWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PeopleWidget</class>
   <extends>QWidget</extends>
   <header>PeopleWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ActivitiesWidget</class>
   <extends>QWidget</extends>
   <header>ActivitiesWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
